.github/workflows/docker.yml

```
@@ -0,0 +1,92 @@
+# GitHub actions workflow which builds and publishes the docker images.
+
+name: Build and deploy docker images
+
+on:
+  push:
+    branches: [ develop, dmr/publish-docker-images ]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  build-sytest-images:
+    name: "Build sytest:${{ matrix.tag }}"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - base_image: ubuntu:bionic
+            tag: bionic
+          - base_image: debian:buster
+            tag: buster
+          - base_image: debian:testing
+            tag: testing
+    steps:
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Inspect builder
+        run: docker buildx inspect
+
+      - name: Log in to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

```suggestion
          password: ${{ secrets.DOCKER_HUB_TOKEN }}
```

There's an org-wide secret which is individually granted to specific repos with this name (if I understand properly)

---

I could see the org-wide secret `DOCKER_HUB_USERNAME` is available to sytest, but I couldn't see one for the token. I was cribbing from [synapse's configuration](https://github.com/matrix-org/synapse/commit/c6509991f362cb559efbb97e1799776cd32a43d8#diff-3f5366f6d6df3ec1179e5efadc6f350bfa88eebf4e2da589b4d94ccb85ae5e94R34-R35)